game/static/game/css/landing.css

```
@@ -893,120 +925,125 @@ body {
-  color: #a0a5b5;
-  font-family: 'Inter', sans-serif;
-  font-size: 1rem;
-  margin-top: -20px;
-  margin-bottom: 48px;
-  max-width: 600px;
-  margin-left: auto;
-  margin-right: auto;
-  line-height: 1.6;
+    text-align: center;
+    color: #a0a5b5;
+    font-family: 'Inter', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)